### ISSUE TITLE
Server can crash upon receiving certain packets.

### ISSUE THREAD

**Issue by AirOne01** [cuberite/cuberite#5322]
Client version: 1.12 ([Wurst](https://github.com/Wurst-Imperium/Wurst7) v6.35.1)
Server OS: Windows 11 Home (v.21H2 build 22000.258)
Cuberite Commit id: d49ce751ba6f8f2eb171633cea30a1119935f8c7

### Expected behavior
Server ignores incoming packets

### Actual behavior
Server crash on certain packets being sent

### Steps to reproduce the behavior
The best way to reproduce this is by using the latest version of the Wurst cheat client.
1. Join your Cuberite server
2. Open the Wurst menu and search for the `LogSpammer` cheat.
![image](https://user-images.githubusercontent.com/21955960/141684815-1223c349-4beb-4387-a96f-5da9d2f8895f.png)
3. Activate it.
4. The server will crash about a second after you enabled the cheat.
![image](https://user-images.githubusercontent.com/21955960/141684837-14ff9b69-6d86-487a-a166-f28f6966280e.png)
If you restart the server and join with the cheat still enabled, it will crash again.

### Server log
```
Info [15:15:42] User AirOne01 authenticated with UUID 4c36470614584778bf6f9254e6e32346
     [15:15:42] A player connected through a proxy without requiring a forwarding secret. If open to the internet, this is very insecure!
Info [15:15:42] Player AirOne01 has joined the game
Warn [15:15:42] cChunkMap::AddEntity: Entity at 0x20cfe7da390 (cPlayer, ID 140) spawning in a non-existent chunk.
     [15:15:43] Unhandled vanilla plugin channel: "MC|PickItem".
Err  [15:15:43] Verification failed: a_ByteBuffer.ReadSome(Message, a_ByteBuffer.GetReadableSpace() - 1), file C:/projects/cuberite/src/Protocol/Protocol_1_8.cpp, line 2883
Err  [15:15:43] Failure report: 

  :(   | Cuberite has encountered an error and needs to close
       | SIGABRT: Server self-terminated due to an internal fault
       |
       | Cuberite AppVeyor (id: 2399)
       | from commit d49ce751ba6f8f2eb171633cea30a1119935f8c7
```

### Additional notes
For this test I only used native plugins (`Core` and `ChatLog`)
I have absolutely no idea if this is too sensitive to be posted here, but since I don't see a lot of projects using Cuberite for production, I'm assuming it's fine.

**Comment by tigerw** [comment]
Hey, could you check to see if it's fixed?

**Comment by AirOne01** [comment]
> Hey, could you check to see if it's fixed?

Fixed on WSL (Ubuntu 20.04.3 LTS)
I will try on Windows as well
